src/Eccube/Controller/Admin/Setting/System/LoginHistoryController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of EC-CUBE
+ *
+ * Copyright(c) EC-CUBE CO.,LTD. All Rights Reserved.
+ *
+ * http://www.ec-cube.co.jp/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Eccube\Controller\Admin\Setting\System;
+
+use Eccube\Controller\AbstractController;
+use Eccube\Form\Type\Admin\SearchLoginHistoryType;
+use Eccube\Repository\LoginHistoryRepository;
+use Eccube\Repository\Master\PageMaxRepository;
+use Eccube\Util\FormUtil;
+use Knp\Component\Pager\Paginator;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * Class LoginHistoryController
+ */
+class LoginHistoryController extends AbstractController
+{
+    /**
+     * @var PageMaxRepository
+     */
+    protected $pageMaxRepository;
+
+    /**
+     * @var LoginHistoryRepository
+     */
+    protected $loginHistoryRepository;
+
+    /**
+     * LoginHistoryController constructor.
+     */
+    public function __construct(
+        PageMaxRepository $pageMaxRepository,
+        LoginHistoryRepository $loginHistoryRepository
+    ) {
+        $this->pageMaxRepository = $pageMaxRepository;
+        $this->loginHistoryRepository = $loginHistoryRepository;
+    }
+
+    /**
+     * ログイン履歴検索画面を表示する.
+     * 左ナビゲーションの選択はGETで遷移する.
+     *
+     * @Route("/%eccube_admin_route%/setting/system/login_history", name="admin_setting_system_login_history")
+     * @Route("/%eccube_admin_route%/setting/system/login_history/{page_no}", requirements={"page_no" = "\d+"}, name="admin_setting_system_login_history_page")
+     * @Template("@admin/Setting/System/login_history.twig")
+     *
+     * @param integer $page_no
+     *
+     * @return \Symfony\Component\HttpFoundation\Response|array
+     */
+    public function index(Request $request, Paginator $paginator, $page_no = null)
```

Comment:
```suggestion
    public function index(Request $request, PaginatorInterface $paginator, $page_no = null)
```

Interface でインジェクションする方が良いかと思います。